docs/install.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2021 Markus Sommer
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+# Installation Instructions
```

Comment:
How about providing a dockerized / Kubernetes option? I'd be interested in working on that.

Reply:
@mogren: I would be very happy about a short documentation and a possible example of use. Thanks a lot for your help.

By the way, there are already two Dockerfiles for dnt7-go. One from @jonashoechst exists for CORE in the [_tutorial_ branch](https://github.com/dtn7/dtn7-go/tree/tutorial/contrib/coreemu). Another one is in the [dtn7-playground repository](https://github.com/dtn7/dtn7-playground). Thus, there should already be some inspiration for the Docker part.